src/include/OSL/oslconfig.h.in

```diff
@@ -118,6 +118,7 @@ using OIIO::TypePoint;
 using OIIO::TypeVector;
 using OIIO::TypeNormal;
 using OIIO::TypeMatrix;
+using OIIO::TypeMatrix44;
```

Comment:
Is there a difference between `TypeMatrix` and `TypeMatrix44` ?

Reply:
It _seems_ that both are the same, in which case maybe its worth consolidating on the former and remove the using declaration for the later?

Reply:
They both mean the same type. I suppose I can remove this and just change all the Matrix44 references to Matrix.

Reply:
updated